wiki/Help_Centre/Client/en.md

```diff
@@ -314,15 +316,15 @@ To make sure osu! can access the internet, we will need to allow it to communica
 
 #### Allowing a program through AVG Firewall
 
-Please consult AVG's [official knowledgebase entry on the topic](https://support.avg.com/SupportArticleView?l=en&urlName=Allow-application-in-AVG-Firewall) for detailed instructions on how to do this.
+Please consult AVG's [official knowledgebase entry on the topic](https://support.avg.com/search?l=en&retUrl=SupportArticleView&searchText=Allow+application+in+AVG+Firewall&supportType=home) for detailed instructions on how to do this.
```

Comment:
I'd rather have it link directly to the article, https://support.avg.com/SupportArticleView?l=en&urlName=AVG-Antivirus-Enhanced-Firewall-Application-Rules&supportType=home